shared/src/business/entities/DocketEntry.ts

```diff
@@ -554,24 +781,53 @@ export class DocketEntry extends JoiValidationEntity {
     return DOCKET_ENTRY_VALIDATION_RULES;
   }
 
-  static isMinuteEntry(rawDocketEntry: RawDocketEntry): boolean {
+  static isMinuteEntry({
+    eventCode,
+    isFileAttached,
+  }: {
+    eventCode: string;
+    isFileAttached?: boolean;
+  }): boolean {
+    if (eventCode === 'RQT') {
+      return !isFileAttached;
+    }
+
     const MINUTE_ENTRIES_EVENT_CODES = Object.keys(MINUTE_ENTRIES_MAP).map(
```

Comment:
I think this could also be 
```suggestion
    const MINUTE_ENTRIES_EVENT_CODES = Object.values(MINUTE_ENTRIES_MAP).map(v => v.eventCode);
```